modules/Bio/EnsEMBL/Compara/Utils/JIRA.pm

```diff
@@ -309,6 +326,18 @@ sub link_tickets {
     my %jira_link_types = map { $_ => 1 } ('After', 'Before', 'Blocks', 'Cloners', 'Duplicate',
                                            'Issue split', 'Related', 'Relates', 'Required');
     if (exists $jira_link_types{$link_type}) {
+        my $inward_ticket = $self->get_ticket($inward_key);
+        my $link_exists = 0;
```

Comment:
Just a comment: in Perl, the result of `my $link_exists;` and this line is exactly the same.